### DISCUSSION TITLE
Priority support terms of Business plan.

### DISCUSSION THREAD

**Discussion by kemeris2000** [lowdefy/lowdefy#1783] [Questions and Help] [answered]
What is Priority support terms of Business plan?

**Comment by Gervwyk** [comment, marked as answer]
Hi @kemeris2000, not the same as community or free plan :) 
Depending on app size, we can schedule call time to advise and help debug, provide some tips etc. 

We're just starting ito support, and we need to establish what limits make business sense. So this will likely change in the future, but for now, we're open to help and advise as much as possible, up to actually writing Lowdefy config for your team, that is where enterprise support starts.

If you like to discuss further, your welcome to email me at gvw _at_ lowdefy.com
